src/dispatch/case/flows.py

```diff
@@ -209,48 +212,8 @@ def case_new_create_flow(
         case=case, db_session=db_session
     )
 
-    if case.assignee:
-        individual_participants.append((case.assignee.individual, None))
-
-    # we create the tactical group
-    direct_participant_emails = [i.email for i, _ in individual_participants]
-
-    indirect_participant_emails = [t.email for t in team_participants]
-    group = group_flows.create_group(
-        subject=case,
-        group_type=GroupType.tactical,
-        group_participants=list(set(direct_participant_emails + indirect_participant_emails)),
-        db_session=db_session,
-    )
-
-    # we create the storage folder
-    storage_members = []
-    if group:
-        storage_members = [group.email]
-
-    # direct add members if not group exists
-    else:
-        storage_members = direct_participant_emails
-
-    case.storage = storage_flows.create_storage(
-        subject=case, storage_members=storage_members, db_session=db_session
-    )
-
-    # we create the investigation document
-    document = document_flows.create_document(
-        subject=case,
-        document_type=DocumentResourceTypes.case,
-        document_template=case.case_type.case_template_document,
-        db_session=db_session,
-    )
-
-    # we update the ticket
-    ticket_flows.update_case_ticket(case=case, db_session=db_session)
-
-    # we update the case document
-    document_flows.update_document(
-        document=document, project_id=case.project.id, db_session=db_session
-    )
+    if create_resources:
+        case_create_resources_flow(db_session=db_session, case_id=case.id)
```

Comment:
I was wondering that but I wasn't sure. It's core to the workflow and if we moved it to `case_create_resources_flow` it would be too binary.  We have the `conversation_target` flag in the Signal, which is our conditional for creating a conversation or not. 

```python
        if conversation_target:
            try:
                # TODO: Refactor conversation creation using conversation_flows module
                conversation = create_conversation(case, conversation_target, db_session)
                conversation_in = ConversationCreate(
```
Also, we should probably use convo flow here or create a new flow for this.

Reply:
I thought the plan was to not create new conversations, but leveraging existing and configured ones in Dispatch for sending notifications and threading them. What am I missing?

Reply:
If we still need to create a conversation and this depends on `individual_participants` and `team_participants`, should these be passed as parameters to `case_create_resources_flow` to avoid fetching them twice?

Reply:
> I thought the plan was to not create new conversations, but leveraging existing and configured ones in Dispatch for sending notifications and threading them. What am I missing?

Only net new (non-deduplicated/snooze) Signals should ever enter the `case_new_create_flow`.

Filtered ones update the existing thread:

```python
    # we don't need to continue if a filter action took place
    if signal_service.filter_signal(db_session=db_session, signal_instance=signal_instance):
        # If the signal was deduplicated, we can assume a case exists,
        # and we need to update the corresponding signal message
        if (
            signal_instance.filter_action == SignalFilterAction.deduplicate
            and signal_instance.case.signal_thread_ts  # noqa
        ):
            update_signal_message(
                db_session=db_session,
                signal_instance=signal_instance,
            )
        return signal_instance
```